TestCases/nonequilibrium/viscwedge/viscwedge.cfg

```diff
@@ -4,8 +4,8 @@
 % Case description: Mach 5 viscous flow over a 10deg wedge                     %
 % Author: C. Garbacz                                                           %
 % Institution: Strathclyde University                                          %
-% Date: 2012.10.07                                                             %
-% File Version 5.0.0 "Raven"                                                   %
+% Date: 2020.11.01                                                             %
+% File Version 7.0.7 "Blackbird"                                               %
```

Comment:
Run a replace all on these versions before merging please.

Reply:
Got it!